### DISCUSSION TITLE
docker-compose with volume deployment

### DISCUSSION THREAD

**Discussion by halindraprakoso** [caprover/caprover#1381] [Q&A] [answered]
hi @githubsaturn, thank you for this project! I have been using caprover for a couple months now and it's very helpful!

I have a question. So, I want to try hasura, while it's available on one click apps, I (probably) can't use it because I need to include the migration/metadata folders that are generated locally, mount them as a volume inside a docker-compose file, and run the hasura image that applies the migrations on server start. How can I adjust this workflow to work with caprover?

**Comment by githubsaturn** [comment, marked as answer]
I don't know Hasura and I'm not sure how it works. But you can do this:
- You can simply copy your files to your server from local (for example using scp)
- Post deployment of Hasura, you can add new mounted directories (using the CapRover dashboard) and point them to the location of the files you copied from your local

**Reply by halindraprakoso** [reply]
I was trying to do what you said but then I realised that it is easier to have a separate db and do the migration remotely via hasura cli. Thank you for the answer anyway😃
